### PR TITLE
State m_launchBall Isn't Reset (Issue #39)

### LINKED ISSUE #39: m_launchBall isn't reset
`m_launchBall` should probably be defaulted within `disabledInit()`.

### PR DESCRIPTION
**Pull request by Tmase13**
Set m_launchBall to false in the disabledInit() method.

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -212,9 +212,11 @@ public void simulationPeriodic() {
   @Override
   public void disabledInit() {
     solenoidReset();
-
+    
+    m_launchBall = false;
     m_raiseLauncherCannon = false;
     m_isInControlPanelMode = false;
+
```
Comment (RyanStyron) [inline src/main/java/frc/robot/Robot.java]: Remove whitespace.

**Review by RyanStyron — CHANGES_REQUESTED** [review]
Nice work, just remove the whitespace on line 219. Other than that, it is all good.

**Comment by RyanStyron** [comment]
Just resolved this change request and merged with upstream changes so that this PR can be closed and merged.

**Review by RyanStyron — APPROVED** [review]

**Comment by CodingKoopa** [comment]
Looks like there's still a merge conflict.

**Comment by RyanStyron** [comment]
Closed without merging.

**Pull request closed without merging**